### ISSUE TITLE
Crash upon opening any piece of media

### ISSUE THREAD

**Issue by ThatOneCalculator** [johanw666/Signal-Android#101]
```
type: crash
osVersion: google/panther/panther:14/AP1A.240505.005/2024051500:user/release-keys
package: org.thoughtcrime.securesms:142002
process: org.thoughtcrime.securesms
processUptime: 113126 + 675 ms
installer: dev.imranr.obtainium

java.lang.IllegalArgumentException: column 'archive_thumbnail_cdn' does not exist. Available columns: [_id, content_type, message_id, transfer_state, data_size, file_name, data_file, cdn_number, remote_location, remote_key, remote_digest, fast_preflight_id, voice_note, borderless, video_gif, width, height, quote, sticker_pack_id, sticker_pack_key, sticker_id, sticker_emoji, blur_hash, transform_properties, display_order, caption, upload_timestamp, remote_incremental_digest, remote_incremental_digest_chunk_size, data_hash_end, archive_cdn, archive_media_name, archive_media_id, type, date_sent, date_received, date_server, thread_id, from_recipient_id, THREAD_RECIPIENT_ID]
	at org.thoughtcrime.securesms.util.rx.RxStore.update(RxStore.kt:42)
	at [[ ↑↑ Inferred subscribe point ↑↑ ]].(:0)
	at [[ ↓↓ Original trace ↓↓ ]].(:0)
	at android.database.AbstractCursor.getColumnIndexOrThrow(AbstractCursor.java:372)
	at org.signal.core.util.CursorUtil.requireInt(CursorUtil.java:21)
	at org.signal.core.util.CursorExtensionsKt.requireInt(CursorExtensions.kt:21)
	at org.thoughtcrime.securesms.database.AttachmentTable.getAttachment(AttachmentTable.kt:1966)
	at org.thoughtcrime.securesms.database.AttachmentTable.getAttachments(AttachmentTable.kt:1402)
	at org.thoughtcrime.securesms.database.MediaTable$MediaRecord$Companion.from(MediaTable.kt:265)
	at org.thoughtcrime.securesms.mediapreview.MediaPreviewRepository.getAttachments$lambda$4(MediaPreviewRepository.kt:66)
	at org.thoughtcrime.securesms.mediapreview.MediaPreviewRepository.$r8$lambda$jItvT5bjeQY5LM-YvP77dU7HmP8(MediaPreviewRepository.kt:0)
	at org.thoughtcrime.securesms.mediapreview.MediaPreviewRepository$$ExternalSyntheticLambda0.call(R8$$SyntheticClass:0)
	at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
	at org.signal.core.util.concurrent.SignalExecutors$NumberedThreadFactory$1.run(SignalExecutors.java:109)
	at [[ ↑↑ Original Trace ↑↑ ]].(:0)
	at [[ ↓↓ Exception Message ↓↓ ]].(:0)
	at column 'archive_thumbnail_cdn' does not exist. Available columns: [_id, content_type, message_id, transfer_state, data_size, file_name, data_file, cdn_number, remote_location, remote_key, remote_digest, fast_preflight_id, voice_note, borderless, video_gif, width, height, quote, sticker_pack_id, sticker_pack_key, sticker_id, sticker_emoji, blur_hash, transform_properties, display_order, caption, upload_timestamp, remote_incremental_digest, remote_incremental_digest_chunk_size, data_hash_end, archive_cdn, archive_media_name, archive_media_id, type, date_sent, date_received, date_server, thread_id, from_recipient_id, THREAD_RECIPIENT_ID].(:0)
```

**Comment by ThatOneCalculator** [comment]
Nvm, uninstalling & reinstalling fixed it

**Comment by johanw666** [comment]
That was a known bug of Signal 7.8.0, fixed in 7.8.1.
